src/seedu/addressbook/AddressBook.java

```diff
@@ -200,7 +200,22 @@
      */
     public static void main(String[] args) {
         showWelcomeMessage();
-        processProgramArgs(args);
+        
```

Comment:
Are methods the only constructs responsible for abstraction? Think about other constructs contributing to SLAP.